spec/content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/used_spec.rb

```diff
@@ -71,12 +71,60 @@
     it_behaves_like "used"
   end
 
+  shared_examples_for "used user quota" do
+    let(:quota_source) { MiqAeMethodService::MiqAeServiceUser.find(@user.id) }
+    let(:quota_source_type) { 'user' }
+    let(:active_method) { 'active_provisions_by_owner'.to_sym }
+    it "check" do
+      @miq_provision_request.options[:owner_email] = owner_email
+      @user.email = requester_email
+      @miq_provision_request.save
+      @user.save
```

Comment:
You can update and save each of these objects in a single line.

```ruby
@miq_provision_request.update(:options => @miq_provision_request.options.merge(:owner_email => owner_email))
@user.update(:email => requester_email)
```

Reply:
Changed